components/VercelChat/tools/ArtistHeroSection.tsx

```diff
@@ -42,7 +44,7 @@ const ArtistHeroSection = ({
             <div className="flex items-center gap-2 mb-3">
               <CheckCircle className="w-4 h-4 text-green-500" />
               <span className="text-green-400 text-sm font-medium">
-                Profile Updated Successfully!
+                {message || "Profile Updated Successfully!"}
```

Comment:
nice work expanding an existing component with a new prop for increased usability across the Recoup app 👏